test/ig/IGExporter.test.ts

```diff
@@ -28,7 +28,36 @@ describe('IGExporter', () => {
       );
       const fixtures = path.join(__dirname, 'fixtures', 'simple-ig');
       const packageJSON: PackageJSON = fs.readJSONSync(path.join(fixtures, 'package.json'));
-      pkg = new Package(packageJSON);
+      const config: Configuration = {
+        filePath: path.join(fixtures, 'config.yml'),
+        id: 'sushi-test',
+        canonical: 'http://hl7.org/fhir/sushi-test',
+        version: '0.1.0',
+        name: 'sushi-test',
+        title: 'FSH Test IG',
+        description: 'Provides a simple example of how FSH can be used to create an IG',
+        dependencies: [
+          { packageId: 'hl7.fhir.r4.core', version: '4.0.1' },
```

Comment:
I noticed this test is skipped now anyway, but if we bring it back, we shouldn't have FHIR R4 listed as a dependency since there is now a dedicate `fhirVersion` property.